tests/pipeline/test_AE.py

```diff
@@ -7,6 +7,7 @@ class Test_AE_Pipeline:
     def pipeline_run(self, demo_dir):
         LOGGER.info("run aberrant expression pipeline...")
         pipeline_run = run(["snakemake", "aberrantExpression", f"-j{CORES}"], demo_dir)
+        tmp = run(["snakemake", "--unlock"], demo_dir)
```

Comment:
Is this a safety measurement when other pipelines fail?

Reply:
I had some issues with the graph execution. Hence I included it here. Not sure if it is really needed but it is for sure not breaking stuff. 😄